smithy-typescript-codegen/src/main/java/software/amazon/smithy/typescript/codegen/PaginationGenerator.java

```diff
@@ -167,57 +160,41 @@ private void writePager() {
         String inputTokenName = paginatedInfo.getPaginatedTrait().getInputToken().get();
         String outputTokenName = paginatedInfo.getPaginatedTrait().getOutputToken().get();
 
-        writer.writeDocs("@public")
+        writer.addImport("createPaginator", null, TypeScriptDependency.SMITHY_CORE);
+
+        writer.writeDocs("@public");
+        writer.write(
+            """
+            export const paginate$L: (config: $LPaginationConfiguration, input: $L, ...rest: any[]) => Paginator<$L> =
+            """,
+            operationName,
+            aggregatedClientName,
+            inputTypeName,
+            outputTypeName
+        );
+        writer
             .openBlock(
```

Comment:
This is a template where it would help to make a Map to contain the variables by name, use a text block without openBlock, and then refer to variables by name in the template. See https://smithy.io/2.0/guides/building-codegen/generating-code.html#named-parameters